projects/control-service/projects/pipelines_control_service/src/integration-test/resources/application-private-builder.properties

```diff
@@ -1,3 +1,4 @@
 datajobs.builder.registrySecret=integration-test-docker-pull-secret
 datajobs.builder.registrySecret.content.testOnly=${BUILDER_TEST_REGISTRY_SECRET}
 datajobs.builder.image=${DOCKER_REGISTRY_URL}/versatiledatakit/job-builder:1.2.3
+datajobs.deployment.dataJobBaseImage=ghcr.io/versatile-data-kit-dev/dp/versatiledatakit/data-job-base-python-3.7:latest
```

Comment:
Maybe we can create a second versatile-data-it-dev-second user ? It needs to be documented and  credentials need be in both Lastpass and Gitlab CI.

Reply:
I think this is a lot of admin overhead for a single test. 
If we come across a second use case where we need to test multiple private repos I will add it in and test this also. 
Does that sound ok with you?